src/controller/ViewReportsScreenController.java

```diff
@@ -34,7 +43,12 @@ public void registerMainApp(MainFXApplication main) {
      * Updates the reports list with stringified reports.
      */
     public void setReportsList() {
```

Comment:
I think we should pass a `Stream<? extends Report>` into this method, then vary the type of report using that.